backend/pkg/api/types/user.go

```diff
@@ -117,25 +117,25 @@ type ExtraDashboardValidatorsPremiumAddon struct {
-	WebhookEndpoints                            uint64              `json:"webhook_endpoints"`
-	MobileAppCustomThemes                       bool                `json:"mobile_app_custom_themes"`
-	MobileAppWidget                             bool                `json:"mobile_app_widget"`
-	MonitorMachines                             uint64              `json:"monitor_machines"`
-	MachineMonitoringHistorySeconds             uint64              `json:"machine_monitoring_history_seconds"`
-	NotificationsMachineCustomThreshold         bool                `json:"notifications_machine_custom_threshold"`
-	NotificationsValidatorDashboardRealTimeMode bool                `json:"notifications_validator_dashboard_real_time_mode"`
-	NotificationsValidatorDashboardGroupOffline bool                `json:"notifications_validator_dashboard_group_offline"`
+	AdFree                                         bool                `json:"ad_free"` // note that this is somhow redunant, since there is already ApiPerks.NoAds
+	ValidatorDashboards                            uint64              `json:"validator_dashboards"`
+	ValidatorsPerDashboard                         uint64              `json:"validators_per_dashboard"`
+	ValidatorGroupsPerDashboard                    uint64              `json:"validator_groups_per_dashboard"`
+	ShareCustomDashboards                          bool                `json:"share_custom_dashboards"`
+	ManageDashboardViaApi                          bool                `json:"manage_dashboard_via_api"`
+	BulkAdding                                     bool                `json:"bulk_adding"`
+	ChartHistorySeconds                            ChartHistorySeconds `json:"chart_history_seconds"`
+	EmailNotificationsPerDay                       uint64              `json:"email_notifications_per_day"`
+	ConfigureNotificationsViaApi                   bool                `json:"configure_notifications_via_api"`
+	ValidatorGroupNotifications                    uint64              `json:"validator_group_notifications"`
+	WebhookEndpoints                               uint64              `json:"webhook_endpoints"`
+	MobileAppCustomThemes                          bool                `json:"mobile_app_custom_themes"`
+	MobileAppWidget                                bool                `json:"mobile_app_widget"`
+	MonitorMachines                                uint64              `json:"monitor_machines"`
+	MachineMonitoringHistorySeconds                uint64              `json:"machine_monitoring_history_seconds"`
+	NotificationsMachineCustomThreshold            bool                `json:"notifications_machine_custom_threshold"`
+	NotificationsValidatorDashboardRealTimeMode    bool                `json:"notifications_validator_dashboard_real_time_mode"`
+	NotificationsValidatorDashboardGroupEfficiency bool                `json:"notifications_validator_dashboard_group_offline"`
```

Comment:
**minor:** Should the json name already be changed here in this PR or later in a frontend task?